### PR TITLE
Include N2V2 into README.md

### PR DESCRIPTION
**Pull request by tibuch**
Hi @alex-krull,

Would these changes be okay with you? 

Best wishes,
Tim-Oliver

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ __Note:__ You can use the N2V2 functionality by providing the following three pa
 * `blurpool=True`, by default set to `False`
 * `skip_skipone=True`, by default set to `False`
 * `n2v_manipulator="median"`, by default set to `"uniform_withCP"`
+* `unet_residual=False`, by default set to
```
Comment (jdeschamps) [inline README.md:79]: Missing default value of the unet_residual (False?)

**Comment by tibuch** [comment]
Should we add a note that we have not tested N2V2 with struct-N2V? This might not work.

**Comment by jdeschamps** [comment]
> Should we add a note that we have not tested N2V2 with struct-N2V? This might not work.

Oh yes, that would make sense for now.

**Comment by tibuch** [comment]
> Another comment: the email address for contact is still alex' @mpi-cbg.de.
> 
> We should ask whether we can update with his new one, we could also add Florian's @fht.org and maybe yours? What do you think?

I think this is the address for corresponding author, which would be Alex. I would say @alex-krull should decided which address should be there.

**Pull request merged**